README.md

```diff
@@ -44,57 +44,48 @@ Istio is composed of these components:
   > simplifies and enhances how microservices in an application talk to each
   > other over the network provided by the underlying platform.
 
-- **Mixer** - Central component that is leveraged by the proxies and microservices
-   to enforce policies such as authorization, rate limits, quotas, authentication, request
-   tracing and telemetry collection.
+- **Istiod** - The Istio control plane. It provides service discovery, configuration and certificate management. It consists of the following sub-components:
 
-- **Pilot** - A component responsible for configuring the proxies at runtime.
+    - **Pilot** - Responsible for configuring the proxies at runtime.
 
-- **Citadel** - A centralized component responsible for certificate issuance and rotation.
+    - **Citadel** - Responsible for certificate issuance and rotation.
 
-- **Citadel Agent** - A per-node component responsible for certificate issuance and rotation.
-
-- **Galley** - Central component for validating, ingesting, aggregating, transforming and distributing config within Istio.
+    - **Galley** - Responsible for validating, ingesting, aggregating, transforming and distributing config within Istio.
```

Comment:
I am not sure this is really accurate (same with calling the CA "citadel") but probably still useful and certainly more accurate than currently

Reply:
Yeah, I wasn't quite sure what the best way to handle those issues was without rewriting the whole thing.

Happy to update with suggestions however.